### ISSUE TITLE
Muffin Module does not dance in Firefox

### ISSUE THREAD

**Issue by botwhytho** [seanttaylor/yangtze#21]
😢 

I am able to run some different Rx demo code so will look into this. Firefox is my daily driver so would want to get this working

**Comment by botwhytho** [comment]
Maybe differently named CSS properties?

**Comment by seanttaylor** [comment]
Yeah. So the `type=module` attribute of the script tag is what identifies the file set on the `src` attribute as an ES6 module rather than a regular old JavaScript file. 

`type=module` is what allows the use of `import` and `export` statements in ES6 modules. Ultimately this attribute was a workaround to allow for distinctions between ES6 modules and modules written according to the CommonJS spec, that is, modules that handle imports using a `require()` statement and exports via the `module.exports` statement. 

Additionally I think this attribute allows browsers that haven't completed the implementation of ES6 modules to use them until said implementation is finished and fully baked in.
